ldap-client_test.go

```diff
@@ -45,6 +45,21 @@ func Test_LDAPClient(t *testing.T) {
 		log.Printf("Groups: %+v", groups)
 	})
 
+	t.Run("GetAllGroupsWithMembersByName", func(t *testing.T) {
```

Comment:
Not sure I understand this test, where do you assert the output?

Reply:
Agreed.
I think a format of "{function-name} - {short-test-description}" could be better.